modules/tetris/ptetris.h

```diff
@@ -0,0 +1,302 @@
+#ifndef PTETRIS_H
+#define PTETRIS_H
+
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+void pt_pixel(int x, int y, int col);
+
+extern uint32_t pt_colors[];
+
+// Max array size
+#ifndef PT_MAX_WIDTH
+	#define PT_MAX_WIDTH 40
```

Comment:
This file uses tabs, the .c uses 4-spaces.  4-spaces is preferred, please convert this file.

Reply:
This is a copied directly from my ptetris project, which uses tabs. So I would have to convert the file to spaces every time I update the file. Not a big deal.